app/views/view_data/delivery_organisations/show.html.erb

```diff
@@ -1,6 +1,6 @@
 <main id="content" role="main">
```

Comment:
Is this page linked to from anywhere at the moment? Not sure if it needs to be yet, just wondered as can't find it unless I go directly to delivery_organisations#show path.

Reply:
No, you need to know the URL. So something like https://service-performance-staging.cloudapps.digital/performance-data/departments/D1198 which is just https://service-performance-staging.cloudapps.digital/performance-data/departments/D1198/metrics/delivery_organisation without the metrics and sort option.  

The links in https://sp-trends.herokuapp.com/a-separate/org-environment-agency (underneath change time period) are where the links will eventually appear. Probably best for us to get the charts and percentage comparison done before we link to it.

Reply:
👍